pkg/kubectl/resource/builder.go

```diff
@@ -532,34 +532,30 @@ func (b *Builder) visitorResult() *Result {
 		if len(b.resources) == 0 {
 			return &Result{singular: isSingular, err: fmt.Errorf("you must provide a resource and a resource name together")}
 		}
-		if len(b.resources) > 1 {
-			return &Result{singular: isSingular, err: fmt.Errorf("you must specify only one resource")}
-		}
 
+		visitors := []Visitor{}
```

Comment:
By tuples I mean the `a/d` form - so treat `a,b,c d` as a transform to `a/d b/d c/d`